cfgov/v1/models/campaign_page.py

```diff
@@ -0,0 +1,53 @@
+from wagtail.admin.edit_handlers import (
+    ObjectList, StreamFieldPanel, TabbedInterface
+)
+from wagtail.core.blocks import StreamBlock, StructBlock
+from wagtail.core.fields import StreamField
+from wagtail.core.models import PageManager
+from wagtail.search import index
+
+from v1 import blocks as v1_blocks
+from v1.atomic_elements import molecules, organisms
+from v1.feeds import FilterableFeedPageMixin
+from v1.models.base import CFGOVPage
+from v1.util.filterable_list import FilterableListMixin
```

Comment:
```suggestion
from wagtail.core.blocks import StreamBlock
from wagtail.core.fields import StreamField
from wagtail.core.models import PageManager

from v1.atomic_elements import molecules, organisms
from v1.models.base import CFGOVPage
```

I think these can go.

Reply:
Yeah, I lazily left in a bunch of stuff I copied in from a page model that I used as an example.